test/TestUtils.py

```diff
@@ -160,23 +161,34 @@ def test_expand_paths_vars(monkeypatch):
 @pytest.mark.parametrize(
     ('reset_env_var', 'message_prefix'),
     (
-        ('PATH', 'Warning: Failed to locate command:'),
-        ('GIT_DIR', 'Warning: Failed to discover yaml files to lint using git:')
+        ('PATH',
+            "Failed to locate command: "),
+        ('GIT_DIR',
+            "Failed to discover yaml files to lint using git: fatal: not a git repository: ''")
     ),
     ids=('no Git installed', 'outside Git repository'),
 )
 def test_get_yaml_files_git_verbose(
     reset_env_var,
     message_prefix,
     monkeypatch,
-    capsys
+    caplog
 ):
     options = cli.get_config(['-v'])
+    utils.initialize_logger(options.verbosity)
     monkeypatch.setenv(reset_env_var, '')
     utils.get_yaml_files(options)
-    msg1, msg2 = capsys.readouterr().out.splitlines()[:2]
-    assert msg1 == 'Discovering files to lint: git ls-files *.yaml *.yml'
-    assert msg2.startswith(message_prefix)
+
+    expected_info = (
+        "ansiblelint",
+        logging.INFO,
+        'Discovering files to lint: git ls-files *.yaml *.yml')
+
+    assert expected_info in caplog.record_tuples
+    for msg in caplog.messages:
+        if msg.startswith(message_prefix):
+            return
+    pytest.fail('%s not found in logged messages: %s', msg, caplog.messages)
```

Comment:
It's better to avoid manual `pytest.fail()` and also it reads better semantically if you do something like "`assert` there's `any` message in captured `messages` that `startswith` `message_prefix`":
```suggestion
    assert any(m.startswith(message_prefix) for m in caplog.messages)
```

Reply:
done